### PR TITLE
Feat/clean up imports, unify config

### PR DESCRIPTION
**Pull request by bartfaimate**

### CODE REVIEW

File: .idea/.gitignore
Comment (dmaniak) [inline .idea/.gitignore:1]: @bartfaimate why do we have 2 .gitignore files?
Edit: is it possible to use the default gitignore in pycharm?

**Review by dmaniak — APPROVED** [review]
Great push, will check rest later 👌

File: .idea/.gitignore
Comment (bartfaimate) [reply .idea/.gitignore:1]: uhh good question
I hope so. Will remove the idea fodler

File: bin/main.py
Comment (dmaniak) [inline bin/main.py:1]: @bartfaimate is it common in python land to place the main file in a `bin` folder? I would have located the built binaries in there

**Review by dmaniak — APPROVED** [review]
@bartfaimate 💯

File: bin/main.py
Comment (bartfaimate) [reply bin/main.py:1]: in python we dont have binaries :D and usually if you do so zou put in `build` folder

**Pull request merged**